src/types.ts

```diff
@@ -24,3 +24,8 @@ export type Value =
 	| "Jack"
 	| "Queen"
 	| "King";
+
+export interface GameListEntry {
+	gameID: number,
+	playerNames: string[]
+}
```

Comment:
It looks like this is only needed in the GameList component, so it should probably live there. You can export it from the index file so that stories and later unit tests can import it if needed. We should probably only put things in this file that we think will be needed across different components and parts of the app.